crates/rpc/src/v03/method/simulate_transaction.rs

```diff
@@ -238,15 +236,6 @@ pub mod dto {
     #[serde_with::serde_as]
     #[derive(Debug, Deserialize, Serialize, Eq, PartialEq)]
     pub struct Event {
```

Comment:
The `Event` type is working only because we're ignoring unknown fields instead of denying them -- the underlying Python type has the `order` property which we don't parse.

Reply:
It is working not "only because" but "exactly because" :) The `order` property is not being propagated anywhere anyway. The whole semantic of it is kind of pointless also - there is always implicit order for each list element.

Reply:
But keeping it distinct and clean seems like a low-hanging fruit. Done.